.rubocop.yml

```diff
@@ -0,0 +1,26 @@
+require:
+  - rubocop-rails
+  - rubocop-rspec
+
+AllCops:
+  NewCops: enable
+  Exclude:
+    - 'db/**/*'
```

Comment:
I would not exclude migration files

Reply:
I think migration files should be excluded .
Since in migrations there are more than 10 lines in the method.